Cargo.toml

```diff
@@ -1,3 +1,23 @@
 [workspace]
 members = ["crates/*"]
 
+[workspace.dependencies]
+qcs-api = "0.2.1"
+qcs-api-client-common = "0.3.0"
+qcs-api-client-grpc = "0.3.0"
+qcs-api-client-openapi = "0.4.0"
+quil-rs = "0.15"
+serde_json = "1.0.86"
+tokio = "1.24.2"
+
+# ndarray is used by the `qcs` crate, but it is also used in the `python` crate via a 
+# re-export through the numpy crate. They should be updated as a pair to keep both 
+# crates version of ndarray in sync.
+# Similarly, pyo3 packages (`numpy`, `pyo3*`) track versions together and need to be
+# updated together.
```

Comment:
Right, will update the comment.

It feels a bit weird to re-export crates `rigetti-pyo3` doesn't use, especially since that crate is open-source and feels like a nice expansion to `pyo3` as is. I wouldn't want to dilute its usefulness by re-exporting a bunch of crates tied to just our use case. I don't love this dependency management either, but it seems to be the reality of working with the pyo3 ecosystem at the moment.

Reply:
I can see an argument for including the re-exports under feature flags, so they aren't dependencies if they aren't being used. I see your point though.